### DISCUSSION TITLE
平白无故出现空白

### DISCUSSION THREAD

**Discussion by Chang-007** [note286/xduts#175] [Q&A] [answered]
编译后这一页顶部出现空白行，不知道是什么原因？
![PixPin_2024-03-13_23-45-35](https://github.com/note286/xduts/assets/55450113/5c57ea67-e06f-4a49-bc70-eb20cc4666f3)

**Comment by note286** [comment, marked as answer]
如果你不能提供复现问题的代码，那么将无法帮助你。
